contrib/asff.tpl

```diff
@@ -18,8 +18,8 @@
     {{- $severity = "INFORMATIONAL" -}}
     {{- end -}}
     {{- $description := escapeString .Description | printf "%q" -}}
-    {{- if gt (len $description ) 1021 -}}
-        {{- $description = (substr 0 1021 $description) | printf "%v .." -}}
+    {{- if gt (len $description ) 1020 -}}
+        {{- $description = (substr 0 1020 $description) | printf "%v ..\"" -}}
```

Comment:
What if we put this truncation before line 20? I think we should not truncate escaped strings because it may break escape. If 1021 is the maximum length of description, we can make it shorter.

Reply:
it was the customer's idea: #1896 
> 1. The description is escaped after truncating. Since escaping adds characters, this can lead to a description longer than the allowed maximum of [1024 characters](https://docs.aws.amazon.com/securityhub/latest/userguide/asff-required-attributes.html#:~:text=Maximum%20length%3A%201%2C024)

i think it makes sense, isn't it?

Reply:
but we can add quotes after the truncation

Reply:
I'm not sure if I correctly understand, but let's say the description is `foo is "bar"`. Then, we escape and double quote it, `"foo is \"bar\""`. If we take 9 bytes, it will be `"foo is \`. Even though we add a quote at last, it will be broken, right? `"foo is \"`. I'm feeling like it is fluky to truncate escaped strings. truncate -> escape looks better.

Reply:
oh, i understood. thanks for explanation.

but we added a space and two dots to a truncated string. at worst we got a strange char at the end...

is escaped string more longer than the raw? it seems it can cause problem.

If you're sure, I'll update the template ` truncate -> escape `

Reply:
>is escaped string more longer than the raw?

I think so because of escape sequences. `truncate` -> `escape` more makes sense, but we should fix this issue as soon as possible. I guess your approach works in most cases. Let me merge it now.

Reply:
Oh, my local GitHub page was stale and it looks like you've not updated yet. Please forget my above comment.

Reply:
@knqyf263 did you see that I've already updated this PR? the new template was merged.

Reply:
Yes, I saw it. Sorry for confusing you.